wit/deps/spin@unversioned/world.wit

```diff
@@ -24,6 +24,7 @@ world platform {
   import mysql;
   import sqlite;
   import redis;
+  import mqtt;
```

Comment:
I think this is probably not needed (I think unversioned is for back compat).  Hopefully someone more knowledgeable can chime in...